src/parameters/toml/ARG2000.toml

```
@@ -0,0 +1,29 @@
+[ARG2000_f_coeff_1]
+value = 0.26583888195264627
+type = "float"
+description = "Scaling coefficient for an empirically determined functional dependence in Abdul-Razzak and Ghan, 2000. Unitless. DOI: https://doi.org/10.1029/1999JD901161"
```

Now only the new value is required for overrides. Up to you!
```suggestion
[ARG2000_f_coeff_1]
value = 0.26583888195264627
```

---

Thats great. I like the short version. We shouldn't duplicate places where we define things.